packages/cli/skills/pikku-middleware/SKILL.md

```
@@ -0,0 +1,232 @@
+
+```typescript
+import { addTagMiddleware } from '.pikku/pikku-types.gen.js'
+
+addTagMiddleware('machine-agent', [machineAgentBearerAuth])
+```
+
+Call at module load time — typically in the same `wirings/*.ts` file as the `wireHTTP` calls that use the tag.
+
+## Middleware Execution Order
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)